gusto/timeloop.py

```diff
@@ -125,5 +147,50 @@ def run(self, t, tmax, pickup=False):
                 state.dump(t, pickup=False)
 
         state.diagnostic_dump()
-
         print "TIMELOOP complete. t= "+str(t-dt)+" tmax="+str(tmax)
```

Comment:
cool - but I might make that a separate issue because it'll make me go through and check whether it'd be useful to have print statements elsewhere for new users